src/Symfony/Component/DomCrawler/UriResolver.php

```diff
@@ -12,22 +12,23 @@
 namespace Symfony\Component\DomCrawler;
 
 /**
- * Expand an URI according a current URI.
+ * The UriResolver class takes an URI (relative, absolute, fragment, etc.)
+ * and turns it into an absolute URI against another given base URI.
  *
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  */
-class UriExpander
+class UriResolver
 {
     /**
-     * Expand an URI according to a current Uri.
+     * Resolve an URI according to a current Uri.
```

Comment:
let's use "base URI" instead of "current URI" everywhere also, to match the RFC?

Reply:
:+1: good idea.